src/main/java/com/amazon/dlic/util/SettingsBasedSSLConfiguratorV5.java

```diff
@@ -51,7 +46,7 @@
 import static org.opensearch.security.ssl.SecureSSLSettings.SSLSetting.SECURITY_SSL_TRANSPORT_KEYSTORE_PASSWORD;
 import static org.opensearch.security.ssl.SecureSSLSettings.SSLSetting.SECURITY_SSL_TRANSPORT_TRUSTSTORE_PASSWORD;
 
-public class SettingsBasedSSLConfiguratorV4 {
+public class SettingsBasedSSLConfiguratorV5 {
```

Comment:
Lets not change the name as I don't think the version number was associated with the Apache Http Client.  What do you think?